staff/david-casas/git/README.md

```diff
@@ -0,0 +1,116 @@
+# GIT
+
+Command interface commands typical in GIT HUB.
+
+## git log (show all the repository commits )
+
+```sh
+$ git log
+commit afeffa743736479982b404b9c647691403420962 (HEAD -> feature/arrays, origin/feature/arrays)
+Author: davidhousesrodriguez <davidhouses@hotmail.com>
+Date:   Fri Feb 16 18:32:40 2024 +0100
+    add new forEach function #197```
+
+## git status (shows the working directory state in wich we are working in )
+
+```sh
+$ git status
+On branch feature/arrays
+Your branch is up to date with 'origin/feature/arrays'.
+nothing to commit, working tree clean
+
```

Comment:
Close each command section with the back ticks, every command should be in a separated back ticks section 🤯

Reply:
Done